src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -228,6 +237,29 @@ export async function POST(request: NextRequest) {
           event,
           updatedSubscriptionFromEvent,
         });
+
+        // get profile id
```

Comment:
I created an `activateProfile` in https://github.com/mozilla/blurts-server/pull/3307 if you want to use that, we could land that first? I think for Foxfooding we need to do that approach anyway.

Reply:
`activateProfile` existed before?